contrib/container-selinux.spec

```diff
@@ -107,11 +107,11 @@ fi
 
 %postun
 if [ $1 -eq 0 ]; then
-%{_sbindir}/semodule -n -r %{modulenames} docker &> /dev/null || :
-if %{_sbindir}/selinuxenabled ; then
-%{_sbindir}/load_policy
-%relabel_files
-fi
+   %selinux_modules_uninstall -s %{selinuxtype} %{modulename} docker
+   if %{_sbindir}/selinuxenabled ; then
```

Comment:
Lines from 111 to 114 are already in the rpm macro.

Reply:
I can't see %relabel_files equivalent in the macro.

Reply:
there is %selinux_relabel_pre and %selinux_relabel_post which has improved functionality and different design